src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertToJsonCommand.cs

```diff
@@ -381,10 +381,8 @@ private void AddSpaces(int numberOfSpacesToReturn, StringBuilder result)
 
         private ErrorRecord NewError()
         {
-            ErrorDetails details = new ErrorDetails(this.GetType().GetTypeInfo().Assembly,
-                "WebCmdletStrings", "JsonStringInBadFormat");
             ErrorRecord errorRecord = new ErrorRecord(
-                new InvalidOperationException(details.Message),
+                new InvalidOperationException(WebCmdletStrings.JsonStringInBadFormat),
```

Comment:
Is the change for the PR? I don't see a description and a test.

Reply:
If the resource file name and the resource string id is known/fixed, there is no point to use `ResourceManager` to retrieve the error message (which is what `ErrorDetails` does). Instead, we can just use the generated C# property.
The existence of the resource string is guaranteed by the build, so I didn't add a test for it.

Reply:
I meant that the PR description mention only `Add-Member` cmdlet but not `ConvertTo-Json`.

Reply:
Updated the description to include the `ConvertTo-Json` change.

Reply:
The error string for this resource (WebCmdletStrings.JsonStringInBadFormat) doesn't seem right:
"The converted JSON string is in bad format."
I think this would be a better error message:
"The JSON string could not be converted because the format is incorrect."

Reply:
This code is from `ConverTo-Json` cmdlet, which takes arbitrary objects and converts them to JSON string. The error message is used when it detects the JSON string we converted to is in bad format.